tests/backup/backup_helper.go

```diff
@@ -43,7 +43,7 @@ const (
 	storkDeploymentName                       = "stork"
 	defaultStorkDeploymentNamespace           = "kube-system"
 	upgradeStorkImage                         = "TARGET_STORK_VERSION"
-	latestStorkImage                          = "openstorage/stork:23.2.0"
```

Comment:
We should not be hardcoding latest stork images here, this would be changing often and we don't want to change code here everytime new stork image is released, I think its better to fetch it from Spec Generator URL like here for example https://install.portworx.com/version